src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
```

Comment:
The class `FruitTransaction` should not start with an empty line. Please remove the empty line to comply with the coding standards.